packages/kbn-es-query/src/es_query/es_query_sql.ts

```diff
@@ -44,3 +44,31 @@ export function getIndexPatternFromSQLQuery(sqlQuery?: string): string {
   }
   return '';
 }
+
+export function getTimeFieldFromTextBasedQuery(textBasedQuery: AggregateQuery): string | undefined {
+  if ('sql' in textBasedQuery) {
+    const query = textBasedQuery.sql?.replaceAll('"', '').replaceAll("'", '');
```

Comment:
why ?

Reply:
Well both methods needs to be re-written but this specific change I did it to support all cases:
- `TIMEPICKER("order_date")`
-  TIMEPICKER('order_date')`
- -  TIMEPICKER(order_date)`